### ISSUE TITLE
[Bug]: `npx create-rslib --template` is broken

### ISSUE THREAD

**Issue by chenjiahan** [web-infra-dev/rslib#426]
### Version

```sh
Rslib 0.0.18
```

### Details

This is expected to work:

```bash
npx create-rslib --template react-js --dir my-lib
```

![Image](https://github.com/user-attachments/assets/8296f0df-42fa-4509-8476-523c5b5556ac)

Ref: 

- https://github.com/rspack-contrib/create-rstack/issues/21
- https://github.com/web-infra-dev/rslib/issues/46#issuecomment-2477890931

### Reproduce link

/

### Reproduce Steps

/

**Comment by fi3ework** [comment]
The template is now generated with optional tools, and the name should be changed to `pnpm create rslib --template "[react]-[]-js" --dir my-lib2`.

Yes, the name is ugly. We might, should support an extra argv for `tools`, for example `pnpm create rslib --template react-js --tool vitest --tool storybook`. What do you think?

**Comment by chenjiahan** [comment]
Agreed, the template name should be clean and intuitive and a new `tool` flag is also helpful.

**Comment by chenjiahan** [comment]
@fi3ework @Timeless0911 Is there any progress on this issue?

**Comment by fi3ework** [comment]
let me check this issue.
